.github/workflows/ci.yaml

```diff
@@ -78,15 +78,20 @@ jobs:
           poetry config virtualenvs.create false          
           python -m venv .venv
           . .venv/bin/activate
-          echo PATH=$PATH >> $GITHUB_ENV
           git clone https://github.com/stackitcloud/stackit-sdk-python-core.git core
```

Comment:
this is still pulling from the archived repo, we can probably remove it right?

Reply:
Oh. You are right. That shouldn't even be there. I didn't even see that